cdhweb/static_src/components/MainNavDesktop/MainNavDesktop.tsx

```diff
@@ -22,6 +24,7 @@ const DesktopMenu = ({
       overview: 'Search our website for people, projects, events or blogs.',
       l2_items: [],
       link_url: searchUrl,
+      is_current: isSearchPage,
```

Comment:
The search nav item isn't part of the json blob, it's hard-coded here. Hence the need to pass in whether the current page is the search page when this component mounts.